_pytest/mark.py

```diff
@@ -168,9 +164,13 @@ def matchkeyword(colitem, keywordexpr):
 
 
 def pytest_configure(config):
-    import pytest
+    config._old_mark_config = MARK_GEN._config
     if config.option.strict:
-        pytest.mark._config = config
+        MARK_GEN._config = config
```

Comment:
@nicoddemus thats a bug i only half-fixed - pytest-unconfigure unsets, pytest_configure sets, whats needed is a better wrapper